dm/config/subtask.go

```diff
@@ -182,13 +182,13 @@ type SubTaskConfig struct {
 	// one go runtime.
 	// IOTotalBytes is used build TCPConnWithIOCounter and UUID is used to as a
 	// key to let MySQL driver to find the right TCPConnWithIOCounter.
-	UUID         string         `toml:"-" json:"-"`
-	IOTotalBytes *atomic.Uint64 `toml:"-" json:"-"`
+	UUID         string         `toml:"uuid" json:"uuid"`
```

Comment:
I remember toml is used for passing task content, and json is used for logging. We don't need to log them?

Also please add unit tests to ensure atomic types has implemented these marshall functions without race

Reply:
> I remember toml is used for passing task content, and json is used for logging. We don't need to log them?

Indeed, we don't need to add json tag for both uuid. json tag used for log and openapi response

Reply:
> Also please add unit tests to ensure atomic types has implemented these marshall functions without race

nice catch! The `atomic.Uint64` values can't be properly serialized to TOML format. How about we manually copy `atomic` value like:https://github.com/pingcap/tiflow/pull/11742/commits/b094cb43c0db9c8dc7b7ae93cff3e4a622ace954

Reply:
since `atomic.Uint64` is not support marshall functions for toml like https://github.com/pingcap/tiflow/blob/master/dm/master/scheduler/worker.go#L282

we need to avoid (nil pointer)/(zero value) after covert `subtaskconfig` to toml... @lance6716  Any suggestion for that?🤔